tests/integration/upsertGraph.js

```diff
@@ -1551,6 +1551,7 @@ module.exports = session => {
 
       after(() => {
         delete Model1.$$jsonSchema;
+        delete Model1.$$validator;
```

Comment:
@koskimas this is the addition that was necessary for the following test to get a new validator.

Reply:
Oh crap! I think I've made the same mistake in other tests too. I'll check them out. Good catch!